exercises/practice/pop-count/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@exercism/javascript-pop-count",
+  "description": "Exercism practice exercise on pop-count",
+  "author": "Katrina Owen",
+  "contributors": [
+    "Derk-Jan Karrenbeld <derk-jan+git@karrenbeld.info> (https://derk-jan.com)",
+    "Tejas Bubane (https://tejasbubane.github.io/)"
+  ],
```

Comment:
@Cool-Katt  Do you want to update these?

Reply:
You mean to add myself as a contributor? I don't know if I properly understand what you mean.

Reply:
Yup. Or as the author. The only person that worked on this exercise is you so I think it ought to be reflected in the author/contributors docs.